.gitignore

```diff
@@ -29,3 +29,7 @@ gradle-app.setting
 **/target/
 **/*.log
 **/nbactions.xml
+
```

Comment:
Sorry, that comment was directed at the overall commit - I'm just seeing it highlight empty-space in the ignore file.

;)